shared/common-adapters/clickable-box.desktop.js

```diff
@@ -31,11 +31,12 @@ const styleFlatButton = {
   transform: 'none',
   textAlign: 'left',
   height: undefined,
-  width: '100%',
+  minWidth: undefined,
   display: 'flex',
   flexDirection: 'column',
   alignItems: 'stretch',
   borderRadius: 0,
+  lineHeight: 0,
```

Comment:
This is a crappy workaround around material-ui. The implementation of this thing is actually a button which has a lineheight which makes it so you can't have a clickable box smaller than a certain size. This overwrites that and makes it 0. This whole thing is going away soon